atest/testdata/libdoc/DataTypesLibrary.json

```diff
@@ -0,0 +1,326 @@
+{
+  "name": "DataTypesLibrary",
+  "doc": "<p>This Library has Data Types.</p>\n<p>It has some in <code>__init__</code> and others in the <a href=\"#Keywords\" class=\"name\">Keywords</a>.</p>\n<p>The DataTypes are the following that should be linked. <span class=\"name\">HttpCredentials</span> , <a href=\"#GeoLocation\" class=\"name\">GeoLocation</a> , <a href=\"#Small\" class=\"name\">Small</a> and <a href=\"#AssertionOperator\" class=\"name\">AssertionOperator</a>.</p>",
+  "version": "",
+  "generated": "2020-11-26 21:27:03",
+  "type": "LIBRARY",
+  "scope": "TEST",
+  "docFormat": "HTML",
+  "source": "/Source/robotframework/robotframework/atest/testdata/libdoc/DataTypesLibrary.py",
```

Comment:
Does this work regardless where Robot's source is checked out?

Reply:
Yes because it is only used to convert to JSON again (roundtrip) and to be converted to XML
And the JSON and XML Spec is used to be read by Libdoc and written to JSON and the schema is verified.

i wanted to have every input ones to test that the output is a valid Schema
And only the dataTypes are really used.